process.md

```diff
@@ -61,126 +62,331 @@ This is for two main reasons:
+   [the mailing list](contact.md#mailing-list).
+2. Bring your design/ideas to [our working group meetings](working-groups.md)
+   for discussion.
+3. Write a [`TEP`](./teps/) from the initial design doc and working group
+   feedback.
 
 A great proposal will include:
 
-* **The use case(s) it solves** Who needs this and why?
-* **Requirements** What needs to be true about the solution?
-* **Alternative proposals** Even if alternatives aren't obvious,
-  forcing yourself to brainstorm a couple more approaches may give you
-  new ideas or make clear that your initial proposal is the best one
+- **The use case(s) it solves** Who needs this and why?
+- **Requirements** What needs to be true about the solution?
+- **Alternative proposals** Even if alternatives aren't obvious, forcing
+  yourself to brainstorm a couple more approaches may give you new ideas or make
+  clear that your initial proposal is the best one
 
 Also feel free to reach out to us on [slack](contact.md#slack) if you want any
 help/guidance.
 
 Thanks so much!!
 
-## OWNERS and Collaborators
+## Contributor Ladder
+
+<!--Big thanks to the folks at https://github.com/cncf/project-template
+for providing the base framework for this section.-->
+
+This contributor ladder outlines the different contributor roles within the
+project, along with the responsibilities and privileges that come with them.
+Community members generally start at the first levels of the "ladder" and
+advance up it as their involvement in the project grows. Our project members are
+happy to help you advance along the contributor ladder.
+
+Each of the contributor roles below is organized into lists of three types of
+things. "Responsibilities" are things that contributor is expected to do.
+"Requirements" are qualifications a person needs to meet to be in that role, and
+"Privileges" are things contributors on that level are entitled to.
+
+### Community Participant
+
+Description: A Community Participant engages with the project and its community,
+contributing their time, thoughts, etc. Community participants are usually users
+who have stopped being anonymous and started being active in project
+discussions.
+
+- Responsibilities:
+  - Must follow the [Tekton CoC](code-of-conduct.md)
+- How users can get involved with the community:
+  - Participating in community discussions ([GitHub, Slack, mailing list, etc](contact.md))
+  - Helping other users
+  - Submitting bug reports
+  - Commenting on issues
+  - Trying out new releases
+  - Attending community events
+
+### Contributor
+
+Description: A Contributor makes direct contributions to the project and adds
+value to it. Contributions need not be code. People at the Contributor level
+may be new contributors, and they can contribute occasionally.
+
+A Contributor must meet the responsibilities of a Community Participant, plus:
+
+- Responsibilities include:
+  - Follow the project [contributing guide](CONTRIBUTING.md)
```

Comment:
since this links to the community contributing guide specifically this might be a bit confusing - for the first time maybe ever i find myself suggesting that we actually remove the link 🤣  and just explicitly say the CONTRIBUTING.md in the repo?